.gitignore

```diff
@@ -28,6 +28,8 @@ dist/bundle.js
 
 # Visual Studio 2015/2017 cache/options directory
 .vs/
+# Visual Studio Code cache/options directory
```

Comment:
I'm not sure if we want this in the gitignore. If VS Code is actually used as one of the main editors, configuration should be synced. You may want to add this to the exclude file in the `.git` directory, which isn't synced (https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file), or actually integrate the VS Code config into the project. The latter would need to be discussed with pongo.

Reply:
you're right, it's not necessary to to have this here.